install_config/aggregate_logging.adoc

```diff
@@ -52,38 +52,24 @@ to manage container logs and prevent filling node disks.
 [[pre-deployment-configuration]]
 == Pre-deployment Configuration
 
-. Ensure that you have  deployed a router for the cluster.
-. Ensure that you have xref:../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[the
+. An Ansible playbook is available to deploy aggregated logging. You should
+familiarize yourself with the
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installation and
```

Comment:
@ewolinetz So in OCP 3.5, is it that `/usr/share/ansible/openshift-ansible/playbooks/common/openshift-cluster/openshift_logging.yml` (that's what this `aggregate_logging.adoc` topic refers to) is used when you're doing a full initial OCP cluster install and you've chosen to deploy logging? And `/usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/openshift-logging.yml` (currently not mentioned in our docs) is what you would use if you wanted to deploy logging for the first time on an existing cluster?

If that's not correct, what's the intended usage of each?